dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -44,3 +46,14 @@ def test_forward_ip():
     with PBSCluster(walltime='00:02:00', processes=4, cores=8, memory='28GB',
                     name='dask-worker') as cluster:
         assert cluster.local_cluster.scheduler.ip == default_ip
+
+
+@pytest.mark.parametrize('qsub_return_string',
+                         ['Request {jobid}.asdf was sumbitted to queue 12.',
+                          '{jobid}',
+                          '  <{jobid}>  ',
+                          '{jobid}; asdf'])
```

Comment:
Nice!

Might I ask you to use complete output as examples provided in https://github.com/dask/dask-jobqueue/issues/132#issuecomment-415306612?

Reply:
Yes, I'll do this.